src/app.jsx

```diff
@@ -125,10 +114,21 @@ class App extends React.Component {
   }
 
   componentDidMount() {
+    this._initData();
     this._initTimeline();
     this._setListeners();
   }
 
+  _initData() {
+    /**
+     * To avoid extra renders before having the data, we set an extra param 'ready' to let the app now when it should initialize.
+     * And we init all collections for the project at this time to avoid asyc issues later
+     */
+    monthDataCollection.fetch().done(_.bind(function(res){
+      this.setState({ ready: true });
+    }, this));
+  }
```

Comment:
In this case, I think we can do the fetch into chart file directly because we are not using this collection out of it, but let's keep it here to have all datai at same place avoiding extra renders.